packages/components/pie-thumbnail/src/thumbnail.scss

```diff
@@ -11,15 +13,26 @@
   border-radius: var(--thumbnail-border-radius);
   border: 1px solid var(--thumbnail-border-color);
   background-color: var(--thumbnail-bg-color);
+  opacity: var(--thumbnail-opacity);
```

Comment:
i think we don't need to set the opacity here and only set it when the component is disabled. 
as by default, the component shouldn't care about opacity until it's needed and browsers will apply the default values anyway. I also think we don't need a variable for it since it's only being used once so for simplicity we can do: 

 &.c-thumbnail--disabled {
    opacity: 50%
  }